docs/packagedelivery.md

```diff
@@ -0,0 +1,11 @@
+# Package Delivery Example
+
+![package delivery sim](packagedeliverysim.gif)
+
+The system under test is the robot moving on the grid, its objective is to delivery all five packages to their goal location (shaded in the same color). The order of delivery does not matter to the robot. Finally, it needs to reach its target location on the left edge of the grid (orange).
+The robot can only carry at maximum one package at once, therefore it cannot move through a state containing a package while it is already carrying another package. It can only drop the packages off at the corresponding target location or back at the pickup location.
```

Comment:
```suggestion
The robot can only carry at maximum one package at once. Therefore, it cannot move through a state containing a package while it is already carrying another package. It can only drop the packages off at the corresponding target location or back at the pickup location.
```